### PR TITLE
Pods 2963

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/controllers/register/trustees/individual/TrusteePhoneController.scala [hunk 1]
```diff
@@ -17,36 +17,67 @@
...
+  private def viewModel(mode: Mode, srn: Option[String], index: Index): Retrieval[CommonFormWithHintViewModel] =
+    Retrieval {
+      implicit request =>
+        TrusteeNameId(index).retrieve.right.map {
+          details =>
+            CommonFormWithHintViewModel(
+              routes.TrusteePhoneController.onSubmit(mode, index, srn),
+              Message("messages__common_phone__heading", Message("messages__common__address_years__trustee").resolve),
+              Message("messages__common_phone__heading", details.fullName),
+              Some(Message("messages__establisher_phone__hint")),
```
Comment (suryabalasingh) [inline app/controllers/register/trustees/individual/TrusteePhoneController.scala]: please rename this message as it says "messages__establisher_phone__hint" but it is applied for trustee

File: test/controllers/register/trustees/individual/TrusteePhoneControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import org.scalatest.mockito.MockitoSugar
+import play.api.data.Form
+import play.api.mvc.Call
+import play.api.test.Helpers._
+import services.FakeUserAnswersService
+import utils.{FakeNavigator, UserAnswers}
+import viewmodels.{CommonFormWithHintViewModel, Message}
+import views.html.phoneNumber
+
+class TrusteePhoneControllerSpec extends ControllerSpecBase with MockitoSugar with BeforeAndAfterEach {
```
Comment (suryabalasingh) [inline test/controllers/register/trustees/individual/TrusteePhoneControllerSpec.scala]: extending MockitoSugar and BeforeAndAfterEach is not required here

File: app/controllers/register/trustees/individual/TrusteePhoneController.scala (hunk 1 above)
Comment (grantrobson) [reply app/controllers/register/trustees/individual/TrusteePhoneController.scala]: Done. Also changed email hint

File: test/controllers/register/trustees/individual/TrusteePhoneControllerSpec.scala (hunk 2 above)
Comment (grantrobson) [reply test/controllers/register/trustees/individual/TrusteePhoneControllerSpec.scala]: done

**Pull request merged**